include/oneapi/dpl/internal/random_impl/lognormal_distribution.h

```diff
@@ -76,7 +76,7 @@ class lognormal_distribution
     void
     param(const param_type& __param)

```

Comment:
Why is the param_type in this class not fixed?

Reply:
We agreed to fix param_type in other classes in another PR

Reply:
We have 7 more distributions. I this PR we change only std::pair to param_type and in the next PR we will fix param_type in other distributions. This is planned to facilitate the review process.